portfolio/src/main/webapp/script.js

```diff
@@ -63,8 +63,21 @@ function previousExperience() {
   selectedExperience--;
 }
 
-function getServerMessage() {
-  fetch('/data').then(response => response.text()).then((message) => {
-    document.getElementById('message-container').innerHTML = message;
+function getServerComments() {
+  fetch('/data').then(response => response.json()).then((comments) => {
+    const individualComments = document.getElementById('comments-container');
+    individualComments.innerHTML = '';
+    individualComments.appendChild(
+      createComment('Comment: ' + comments[0]));
+    individualComments.appendChild(
+      createComment('Comment: ' + comments[1]));
+    individualComments.appendChild(
+      createComment('Comment: ' + comments[2]));
```

Comment:
Please move this code out into a separate function that takes care to populate the relevant HTML elements rather than have all of the logic live inside of this lambda (for legibility)

Reply:
that makes sense, thanks for the feedback